inject-groovy/src/main/groovy/io/micronaut/ast/groovy/visitor/LoadedVisitor.groovy

```diff
@@ -94,6 +95,7 @@ class LoadedVisitor implements Ordered {
         return true
     }
 
+    @CompileDynamic
```

Comment:
without `@CompilieDynamic`

```
 [Static type checking] - Non-static method java.lang.Object#hashCode cannot be called from static context
 @ line 99, column 16.
           return visitor.getClass().hashCode()
```

Reply:
Seems like a regression in Groovy

Reply:
it is present since 4.0.7

Reply:
I submitted an issue https://issues.apache.org/jira/browse/GROOVY-10939